tests/Unit/Evolution/DiscontinuousGalerkin/SlopeLimiters/Test_WenoHelpers.cpp

```diff
@@ -82,11 +97,215 @@ void test_oscillation_indicator_3d() noexcept {
   CHECK(indicator == approx(expected));
 }
 
+struct ScalarTag : db::SimpleTag {
+  using type = Scalar<DataVector>;
+  static std::string name() noexcept { return "Scalar"; }
+};
+
+template <size_t VolumeDim>
+struct VectorTag : db::SimpleTag {
+  using type = tnsr::I<DataVector, VolumeDim>;
+  static std::string name() noexcept { return "Vector"; }
+};
+
+void test_reconstruction_1d() noexcept {
+  const double neighbor_linear_weight = 0.001;
+  const auto mesh = Mesh<1>({{5}}, Spectral::Basis::Legendre,
```

Comment:
Is testing with different number of grid points not possible? I realize in 3d this might be too expensive, but in 1d and probably 2d it should be fine.

Reply:
Possible, but requires a new TEST_CASE for each resolution, for the same caching-related reasons given to justify why these TEST_CASES aren't merged.